Field/styles.scss

```diff
@@ -183,6 +183,18 @@
     @include typography(map-get($font-sizes, big-body-desktop), semi-bold);
   }
 
+  &::-webkit-input-placeholder {
```

Comment:
It's unfortunately not possible to chain this selector with `:-ms-input-placeholder`. That's why they're identical.